src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -125,71 +95,137 @@ public JPanel getOptionsPanel() {
 
     @Override
     public boolean processQuery(String query, ImportInspector dialog, OutputPrinter status) {
-        terms = query;
+        //IEEE API seems to use .QT. as a marker for the quotes for exact phrase searching
+        terms = query.replaceAll("\"", "\\.QT\\.");
+
         shouldContinue = true;
         parsed = 0;
-        unparseable = 0;
-        piv = 0;
         int pageNumber = 1;
 
-        searchUrl = makeUrl(pageNumber);//start at page 1
+        postData = makeSearchPostRequestPayload(pageNumber);
 
         try {
-            URL url = new URL(searchUrl);
-            String page = Util.getResults(url);
+            URL url = new URL(IEEEXploreFetcher.URL_SEARCH);
 
-            if (page.contains("You have entered an invalid search")) {
-                status.showMessage(Localization.lang("You have entered an invalid search '%0'.", terms),
-                        DIALOG_TITLE, JOptionPane.INFORMATION_MESSAGE);
-                return false;
-            }
+            HttpURLConnection con = (HttpURLConnection) url.openConnection();
+
+            //add request header
+            con.setRequestMethod("POST");
+            con.setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 5.1; rv:31.0) Gecko/20100101 Firefox/31.0");
```

Comment:
I have seen this code in the Util, maybe create a util method for opening a url with the default user agent?